cypress/objects/t2Page.js

```diff
@@ -0,0 +1,66 @@
+class T2Page {
```

Comment:
This file should be name `timeLoggingPage `or something like that, because T2 doesn't say much :)